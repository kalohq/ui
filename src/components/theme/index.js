/* @flow */
export {default} from './theme';
