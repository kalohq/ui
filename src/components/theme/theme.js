/* @flow */

import tokens from '../../design-tokens/tokens.theme.js';

/**
 * ThemeProvider Kalo Theme
 * 
 * This is the core CSS-in-JS theme that is passed in to the
 * global ThemeProvider. It is mainly composed of our core
 * design tokens.
 */

const theme = {
  space: [0, 8, 16, 32, 64],
  breakpoints: [],
  ...tokens,
};

export default theme;
