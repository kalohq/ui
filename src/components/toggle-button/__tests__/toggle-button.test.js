/* @flow */
/* eslint-env jest */
import React from 'react';
import renderer from 'react-test-renderer';
import {ThemeProvider} from 'emotion-theming';

import ToggleButton from '../toggle-button.react';
import theme from '../../theme';

describe('ToggleButton', () => {
  test('should render shallow component ok', () => {
    const element = renderer
      .create(
        <ThemeProvider theme={theme}>
          <ToggleButton onChange={() => {}} />
        </ThemeProvider>
      )
      .toJSON();
    expect(element).toMatchSnapshot();
  });
});
