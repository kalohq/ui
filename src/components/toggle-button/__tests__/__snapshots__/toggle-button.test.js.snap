// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ToggleButton should render shallow component ok 1`] = `
<div
  className="e1q9f2xp0 css-f9caso"
  id="toggle-button"
  onClick={[Function]}
  style={Object {}}
>
  <div
    className="css-1mfdy6w e1mw9c3b0"
    selected={false}
  >
    <div
      className="css-1csuntb e1mw9c3b1"
      selected={false}
    />
  </div>
</div>
`;
